README.md

```diff
@@ -1,26 +1,98 @@
 # Loop Exercises
 
 1. Write a `while` loop and a `for` loop that takes the variable "num" and logs all the numbers, descending, between "num" and 1.
-2. Write a `while` loop and a `for` loop that takes the variable "num", and iterates over all numbers from 0 to "num". 
+
+let num = 10
+while (num >= 1) {
+  console.log('the number is: ' + num)
+  num -= 1
+}
+
+
+2. Write a `while` loop and a `for` loop that takes the variable "num", and iterates over all numbers from 0 to "num".
 For each iteration, it will check if the current number is even or odd, and log that to the screen (e.g. "2 is even")
-3. Write a `while` loop and a `for` loop that takes the variable "num" and iterates over all numbers from 0 to "num". 
+
+num = 10
+for (let i = 0; i <= num; i++ )
+  if (i % 2 === 0 ) {
+  console.log( i + " is even" )
+} else {
+console.log(i + 'is odd') }
+
+
+3. Write a `while` loop and a `for` loop that takes the variable "num" and iterates over all numbers from 0 to "num".
 For each iteration of the loop, it will multiply the number by 9 and log the result (e.g. "2 * 9 = 18").
 
-_Bonus_ think of another way to solve it. 
+_Bonus_ think of another way to solve it.
   <details>
     <summary>
       Hint
     </summary>
     Find the final number and increment the loop by 9.
   </details>
-  
+
+  let num = 0
+  while (num <= 10) {
+    console.log(num * 9)
```

Comment:
Make sure that you don't just log the number - log the math, too! Like so: "2 * 9 = 18"